src/plugins/visualizations/public/index.ts

```diff
@@ -31,38 +30,27 @@ export function plugin(initializerContext: PluginInitializerContext) {
 export { Vis } from './vis';
 export { TypesService } from './vis_types/types_service';
 export { VISUALIZE_EMBEDDABLE_TYPE, VIS_EVENT_TO_TRIGGER } from './embeddable';
-export { VisualizationContainer, VisualizationNoResults } from './components';
-export { getSchemas as getVisSchemas } from './legacy/build_pipeline';
+export { VisualizationContainer } from './components';
+export { getVisSchemas } from './vis_schemas';
 
 /** @public types */
 export { VisualizationsSetup, VisualizationsStart };
 export { VisGroups } from './vis_types';
-export type {
-  VisTypeAlias,
-  VisType,
-  BaseVisTypeOptions,
-  ReactVisTypeOptions,
-  Schema,
-  ISchemas,
-} from './vis_types';
+export type { VisTypeAlias, VisTypeDefinition, Schema, ISchemas } from './vis_types';
 export { VisParams, SerializedVis, SerializedVisData, VisData } from './vis';
 export type VisualizeEmbeddableFactoryContract = PublicContract<VisualizeEmbeddableFactory>;
 export type VisualizeEmbeddableContract = PublicContract<VisualizeEmbeddable>;
 export { VisualizeInput } from './embeddable';
-export type ExprVis = ExprVisClass;
-export { SchemaConfig, BuildPipelineParams } from './legacy/build_pipeline';
-// @ts-ignore
+export { SchemaConfig } from './vis_schemas';
```

Comment:
This can be combined with `getVisSchemas `

Reply:
if you take a look at the file, the exports are split with `static` code and `types`.

![image](https://user-images.githubusercontent.com/31325372/105063905-56ea0d00-5a8d-11eb-8e69-4240344201b5.png)

That's why I made an explicit type export here. I'm not sure about the benefit of such an approach..
So maybe we could unify all of them, especially since I see couple of static exports in types section 🤷🏻‍♂️

Reply:
Oh I see, yep I agree but let's not do this on this PR. Wdyt?